_maps/map_files/rift/rift-05-surface2.dmm

```diff
@@ -22244,6 +22276,16 @@
 	pixel_y = 4
 	},
 /obj/item/gunbox/lethal,
+/obj/item/gunbox/lethal{
+	pixel_y = -4
+	},
+/obj/item/gunbox/lethal{
+	pixel_y = 8
+	},
+/obj/item/gunbox/lethal{
+	pixel_y = 4
+	},
+/obj/item/gunbox/lethal,
```

Comment:
Your PR description says you want to add less-than-lethal boxes. If so, remove the `/lethal`:
```suggestion
/obj/item/gunbox{
	pixel_y = -4
	},
/obj/item/gunbox{
	pixel_y = 8
	},
/obj/item/gunbox{
	pixel_y = 4
	},
/obj/item/gunbox,
```